src/app/public/src/modules/wrapper/title.service.ts

```diff
@@ -14,7 +14,7 @@ export class StacheTitleService {
 
     if (parts && parts.length > 0) {
       parts.push(windowTitle);
-      let validParts = parts.filter(part => part !== undefined);
+      let validParts = parts.filter(part => part);
```

Comment:
I think we would want to make this a little more verbose for readability. It's not immediately apparent why we would be trying to call a filter and just return the part being passed in.  
```
parts.filter(part => (part !== undefined && part !== ''))
```
thoughts? @blackbaud-brandonhare @Blackbaud-SteveBrush ?

Reply:
Yeah, I agree; it's a bit difficult to gauge the purpose of the `filter` at first glance.

Reply:
This was the only change I wasn't super sure about. I think that we can be comfortable saying that anything falsey shouldn't make it through the filter. Adding more conditions just makes the filter more complex, and could potentially let through falsey values we aren't expecting in the future. I would be more comfortable adding a comment explaining the filter, in this instance. Let me know your thoughts!